types/dynalite.d.ts

```diff
@@ -10,11 +10,11 @@ declare module "dynalite" {
     maxItemSizeKb?: number;
   }
 
-  export declare class DynaliteServer extends Server {
-    public close(cb?: (err?: Error) => void): void;
+  export class DynaliteServer extends Server {
+    public close(cb?: (err?: Error) => void): this;
```

Comment:
Interesting. Not sure how we can override this, because `close()` doesn't return `this` in `dynalte`: https://github.com/mhart/dynalite/blob/master/index.js#L37